Makefile

```diff
@@ -15,6 +15,13 @@ include ./make/test.mk
 include ./make/docker.mk
 include ./make/csv.mk
 
+BUILD_DIR ?= ./out/
+BUILD_OUT_FILE ?= operator
+VENDOR_BUILD ?= 
+
+.PHONY: build
```

Comment:
did you mean `build_prod` ?

Reply:
@sthaha I've updated the the patch. PTAL.